pymess/models/dialer.py

```diff
@@ -58,6 +58,8 @@ class Meta(BaseMessage.Meta):
 class DialerMessage(AbstractDialerMessage):
 
     is_final_state = models.BooleanField(verbose_name=_('is final state'), null=False, default=False)
+    is_predictive = models.BooleanField(verbose_name=_('is predictive dialer'), null=False, default=False)
+    content = models.TextField(verbose_name=_('content'), null=True, blank=True)
```

Comment:
ty přetěžuješ předka, chápu, je ale divné že půlka věcí je přetížená v předkovi a půlka ne. Dej vše do předka jak is_automatic tak ten nepovinný content a dej tu validaci na model

Reply:
a validation error

na backendu to máš v předkovi a na modelu ne to mi vadí..

Reply:
tohle stále platí

Reply:
fixed